pyttsx3/drivers/sapi5.py

```diff
@@ -88,7 +98,40 @@ def save_to_file(self, text, filename):
 
     @staticmethod
     def _toVoice(attr):
-        return Voice(attr.Id, attr.GetDescription())
+        # Retrieve voice ID and description (name)
+        voice_id = attr.Id
+        voice_name = attr.GetDescription()
+
+        # Retrieve and convert language code
+        language_attr = attr.GetAttribute("Language")
+        language_code = int(language_attr, 16)
+        primary_sub_code = f"{language_code & 0x3FF}-{(language_code >> 10) & 0x3FF}"
+        languages = [lcid_to_locale(primary_sub_code)]
+
+        # Retrieve gender
+        gender_attr = attr.GetAttribute("Gender")
+        gender = (
+            "Male"
+            if gender_attr.lower() == "male"
+            else "Female"
+            if gender_attr.lower() == "female"
+            else None
+        )
```

Comment:
```suggestion
        gender_title_case = gender.title()
        gender = gender_title_case if gender_title_case in {"Male", "Female"} else None
```

Reply:
That was ruff making it like that. I found it wierd myself.